hawc/apps/myuser/management/commands/scrub_db.py

```diff
@@ -33,6 +34,13 @@ def handle(self, *args, **options):
             if input("".join(message)) != "yes":
                 raise CommandError("Scrubbing user data cancelled.")
 
+        self.update_site()
+        self.update_users()
+
+    def update_site(self):
+        Site.objects.update(domain="127.0.0.1:8001", name="localhost")
```

Comment:
should this be 8000 instead of 8001? I think it's the default case

Reply:
oh, yeah, wow good catch!

Reply:
okay, changed it to 8000! all good now